sequencer/src/lib.rs

```diff
@@ -946,7 +946,7 @@ pub mod testing {
                     let epoch_start_block = self.config.epoch_start_block;
 
                     let (genesis_state, genesis_stake) = light_client_genesis_from_stake_table(
-                        &self.config.known_nodes_with_stake,
+                        &self.config.known_nodes_with_stake.clone().into(),
```

Comment:
similarly, what about `config.hotshot_stake_table()`?
(pls do a search and replace, i won't comment on every such pattern)

Reply:
done in [ae5454d](https://github.com/EspressoSystems/espresso-network/pull/3233/commits/ae5454dcb5ba9c67dc7aeda32459284df531c5d4)